cpp/modules/deck.gl/layers/src/line-layer/line-layer.cpp

```diff
@@ -163,7 +160,7 @@ auto LineLayer::getSourcePositionData(const std::shared_ptr<arrow::Table>& table
     throw std::logic_error("Invalid layer properties");
   }
 
-  return ArrowMapper::mapVector3DoubleColumn(table, props->getSourcePosition);
+  return ArrowMapper::mapVector3FloatColumn(table, props->getSourcePosition);
```

Comment:
This is fine for now, but eventually we are going to need to split doubles into two floats...

D1 D2 D3 => F1high F1low F2high F2 low...